src/openforms/utils/json_logic/descriptions.py

```diff
@@ -323,15 +323,45 @@ def op_today(*args, **kwargs) -> str:
 
 @add_boilerplate()
 def op_rdelta(*args) -> str:
-    if len(args) == 1:
-        return gettext("{years} year(s)").format(years=args[0])
-    elif len(args) == 2:
-        return gettext("{years} year(s), {months} month(s)").format(
-            years=args[0],
-            months=args[1],
-        )
-    return gettext("{years} year(s), {months} month(s), {days} day(s)").format(
-        years=args[0],
-        months=args[1],
-        days=args[2],
-    )
+    match len(args):
+        case 1:
+            return gettext("{years} year(s)").format(years=args[0])
+        case 2:
+            return gettext("{years} year(s), {months} month(s)").format(
+                years=args[0],
+                months=args[1],
+            )
+        case 3:
+            return gettext("{years} year(s), {months} month(s), {days} day(s)").format(
+                years=args[0],
+                months=args[1],
+                days=args[2],
+            )
+        case 4:
+            return gettext("{years}y, {months}m, {days}d, {hours}h").format(
+                years=args[0],
+                months=args[1],
+                days=args[2],
+                hours=args[3],
+            )
+        case 5:
+            return gettext(
+                "{years}y, {months}m, {days}d, {hours}h, {minutes}min"
+            ).format(
+                years=args[0],
+                months=args[1],
+                days=args[2],
+                hours=args[3],
+                minutes=args[4],
+            )
+        case 6:
+            return gettext(
+                "{years}y, {months}m, {days}d, {hours}h, {minutes}min, {seconds}s"
+            ).format(
+                years=args[0],
+                months=args[1],
+                days=args[2],
+                hours=args[3],
+                minutes=args[4],
+                seconds=args[5],
+            )
```

Comment:
Matching can unpack too:
```suggestion
    match args:
        case [years]:
            return gettext("{years} year(s)").format(**locals())
        case [years, months]:
            return gettext("{years} year(s), {months} month(s)").format(**locals())
        case [years, months, days]:
            return gettext("{years} year(s), {months} month(s), {days} day(s)").format(**locals())
        case [years, months, days, hours]:
            return gettext("{years}y, {months}m, {days}d, {hours}h").format(**locals())
        case [years, months, days, hours, minutes]:
            return gettext(
                "{years}y, {months}m, {days}d, {hours}h, {minutes}min"
            ).format(**locals())
        case [years, months, days, hours, minutes, seconds]:
            return gettext(
                "{years}y, {months}m, {days}d, {hours}h, {minutes}min, {seconds}s"
            ).format(**locals())
        case _:
            raise ValueError("Unexpected amount of arguments. Expected 1-6 got {len(args)}: {args!r}")

```
Should work... 

The error is really needed. If I had to write jsonlogic, I'd want all the help I could get from a compiler.